docs/software/spack.md

```diff
@@ -143,7 +143,7 @@ In addition to standard software packages, you can use Spack to install compiler
     spack install --add gcc@13.1.0
     ```
```

Comment:
Can you replace these "or" examples with tabs? 

![Screenshot 2024-09-20 at 10 47 57 AM](https://github.com/user-attachments/assets/06714c93-5d7d-43e6-a359-2fc85b2aed73)

Reply:
I’ve added the tabs, but I'm not entirely convinced about the titles for the tabs. Do you think they’re okay?

Reply:
Yup those are ok!